scripts/components/Main.js

```diff
@@ -521,6 +523,15 @@ export default class Main extends React.Component {
     this.state.scoreCard.sceneScoreCards[sceneId].scores[assignmentId] = score;
   }
 
+  updateEscapeScoreCard(isUnlocked){
+    this.state.scoreCard.totalCodesEntered = this.state.scoreCard.totalCodesEntered + 1;
+
+    if(isUnlocked){
+      this.state.scoreCard.totalCodesUnlocked = this.state.scoreCard.totalCodesUnlocked + 1;
+    }
```

Comment:
Use setState when changing component state to trigger re-render.

```suggestion
    const totalCodesEntered = this.state.scoreCard.totalCodesEntered + 1;
	const totalCodesUnlocked = this.state.scoreCard.totalCodesUnlocked + (isUnlocked ? 1 : 0);

    this.setState({ 
      scoreCard: {
        ...this.state.scoreCard,
        totalCodesEntered,
        totalCodesUnlocked,
      }
    });
```